shopfloor/actions/search.py

```diff
@@ -169,11 +169,12 @@ def generic_packaging_from_scan(self, barcode):
         )
 
     def origin_move_from_scan(self, barcode, extra_domain=None):
+        origins = self.env["delivery.carrier"]._get_origin_from_barcode(barcode)
```

Comment:
```suggestion
    def origin_move_from_scan(self, barcode, extra_domain=None, origins=None):
        origins = origins or [barcode]
```
And move `self.env["delivery.carrier"]._get_origin_from_barcode(barcode)` to the override in sf_reception.

Reply:
I don't like this adding args because we expect smth that's impremented in another module thing.
I think we should either we move this whole "origin_move_from_scan" method in sf_reception or keep it like this.

Reply:
Then delegate it to an hookable method :man_shrugging: 
Whatever you like most but don't leave it here :smile:

Reply:
We can't call `self.env["delivery.carrier"]._get_origin_from_barcode(barcode)` from `shopfloor` anyway unless to add a dependency on `delivery_carrier_return_barcode` in `shopfloor`

Reply:
yeah, added a hook, forgot to change the call.